splunklogger.js

```diff
@@ -14,7 +14,7 @@
  * under the License.
  */
 
-var request = require("request");
+var needle = require("needle");
```

Comment:
- [ ] Let's add an item to the CHANGELOG.md for this - under `New features & APIs`: Replaced client implementation to use `needle` in lieu of deprecated `request`

Reply:
Updated CHANGELOG.md file as suggested.